<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>661</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Забронировать столик</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>51</y>
        <width>641</width>
        <height>441</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>10</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Поиск</string>
      </property>
     </widget>
     <widget class="QPushButton" name="searchButton">
      <property name="geometry">
       <rect>
        <x>219</x>
        <y>10</y>
        <width>50</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Найти</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab2">
     <attribute name="title">
      <string>Активные брони</string>
     </attribute>
     <widget class="QListWidget" name="reservationsList">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>651</width>
        <height>491</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="spacing">
       <number>10</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="logoutButton">
    <property name="geometry">
     <rect>
      <x>612</x>
      <y>20</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Выйти</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>434</x>
      <y>30</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
